openhands/server/routes/conversation.py

```diff
@@ -205,7 +207,12 @@ async def get_microagents(
         for name, agent in memory.repo_microagents.items():
             microagents.append(
                 MicroagentResponse(
-                    name=name, type='repo', content=agent.content, triggers=[]
+                    name=name, 
+                    type='repo', 
+                    content=agent.content, 
+                    triggers=[],
+                    inputs=getattr(agent, 'inputs', []),
+                    tools=getattr(agent, 'tools', [])
```

Comment:
@OpenHands I think the field should be .mcp_tools - can you look around source code to confirm